src/data/python-quiz.ts

```diff
@@ -1143,6 +1143,16 @@ const pythonQuiz = [
     Explanation: "+ is used to concatenate and * is used to multiply strings", 
     Link: "https://www.freecodecamp.org/news/learn-python-by-thinking-in-types", 
   },
+  {
+    Question: "Which sorting algorithm is used for built-in sort method of Python?",
+    Answer: "Tim Sort",
+    Distractor1: "Quick Sort",
+    Distractor2: "Merge Sort",
+    Distractor3: "Tim Sort",
+    Distractor4: "Insertion Sort",
+    Explanation: "Python's default sort uses Tim Sort, which is a combination of both merge sort and insertion sort.",
```

Comment:
```suggestion
    Explanation: "Python's Tim Sort algorithm is a combination of merge and insertion sort.",
```

Reply:
Thanks, @jdwilkin4. I just updated this PR with changes!